src/types/experience/ExperienceService.ts

```diff
@@ -13,17 +13,8 @@ governing permissions and limitations under the License.
 import { Experience, ExperienceField } from "./Experience";
 
 import { GuestUI } from "@adobe/uix-guest";
-import { VirtualApi } from "@adobe/uix-core";
 import { GenerationContext } from "../generationContext/GenerationContext";
-
-export interface RightPanelApi extends VirtualApi {
-  api: {
-    createRightPanel: {
-      getExperiences: () => Promise<any[]>;
-      getGenerationContext: () => Promise<any>;
-    };
-  };
-}
+import { RightPanelApi } from "../RightPanel/RightPanel";
 
 export class ExperienceError extends Error {
   constructor(message: string) {
```

Comment:
keeping experienceError and experienceService for backwards compatibility but can remove in future iterations